backend/uclapi/oauth/tests.py

```diff
@@ -758,6 +758,121 @@ def test_auto_accept_shibcallback_real_account(self):
         )
         self.assertEqual(response.status_code, 302)
 
+    def test_userallow_no_post_data(self):
+        response = self.client.get(
+            '/oauth/user/allow',
+            {}
+        )
+        self.assertEqual(response.status_code, 400)
+        self.assertEqual(response.json()["error"],
+                         ("The signed data received "
+                          "was invalid."
+                          " Please try the login process again."
+                          " If this issue persists, please contact support."))
```

Comment:
What support? IMO all error strings should be telling them to contact us (or ISD?).

Reply:
Yes they should probably all say email isd.apiteam@ucl.ac.uk , I will create an issue for that.